styling-and-themes/overview.md

```diff
@@ -2,198 +2,131 @@
 title: Built-in Themes
 page_title: Themes
 description: The UI for Blazor suite comes with a set of built-in themes that you can choose from. Bootstrap and Material themes are also included.
-slug: general-information/themes
+slug: themes-built-in
 tags: telerik,blazor,theme,built-in
 published: True
-previous_url: /themes/overview
+previous_url: /themes/overview,/styling-and-themes/form-elements
 position: 0
 ---
 
 # Built-in Themes
 
-Telerik UI for Blazor comes with a set of built-in themes that you can choose from to alter the visual appearance of our components. You can compare all themes on the [Telerik UI for Blazor live demos](https://demos.telerik.com/blazor-ui/)).
+Telerik UI for Blazor comes with a set of built-in CSS themes that control the visual appearance of the components. Each theme determines the components' [colors](https://www.telerik.com/design-system/docs/foundation/color/), borders, backgrounds, size, layout, position, font size and sometimes the font family. You can compare all themes and swatches on the [Telerik UI for Blazor live demos](https://demos.telerik.com/blazor-ui/).
+
+[Telerik UI for Blazor has the same HTML rendering and theme stylesheets as other Telerik and Kendo UI web products](#built-in-theme-development), so previous experience with them can be helpful. At the same time, the Telerik UI for Blazor components are *native Blazor components* and not wrappers around other Telerik products.
+
+## Theme Names
 
 The available built-in themes are:
 
-* **Default** is our own neutral styling that suits most cases. The theme has a [swatch (color variation)]({%slug themes-swatches%}) called [Ocean Blue]({%slug themes-accessibility-swatch%}), which features enhanced accessibility.
-* **Bootstrap** matches the styling of the [Bootstrap CSS framework](https://getbootstrap.com). Read more in the [Bootstrap Notes](#bootstrap-notes) section below.
-* **Material** implements the [Material Design Guidelines](https://material.io/design/). Read more in the [Material Notes](#material-notes) section below.
+* **Default** uses a neutral Telerik design and suits most cases. The theme has a built-in [swatch (color variation)]({%slug themes-swatches%}) called [Ocean Blue A11y]({%slug themes-accessibility-swatch%}), which provides enhanced contrast for WCAG level AA accessibility compliance.
```

Comment:
Use italics instead of bold for the theme names when you introduce them as new terms.

Reply:
I discussed this with Iva in advance and we agreed to leave this discrepancy like this for better readability.